src/backend/distributed/citus--7.2-3--7.2-4.sql

```diff
@@ -0,0 +1,6 @@
+/* citus--7.2-2--7.2-3 */
+
+DROP FUNCTION IF EXISTS master_expire_table_cache(regclass);
```

Comment:
Be careful with these. We messed up before by using the wrong signature and due to the IF EXISTS this is silently ignored.
E.g. Does this actually drop the function without the pg_catalog prefix?

You may also want to remove the original create function from the SQL file, such that we could safely remove the C definition.